README.md

```diff
@@ -19,18 +19,42 @@ as `abc` version `X.Y.Z+dune`.
 
 ## Port a new package/version to dune on `opam-overlays`
 
-The typical workflow to port an existing package to dune and publish it on
-`opam-overlays` would be the following:
-
-1. Create a fork of the upstream package `abc`
-2. Port it to dune on a specific branch `dune-universe-vX.Y.Z`, where `X.Y.Z`
-   corresponds to the upstream version you're dunifying. Make sure the opam file
-   has been properly updated and that dune is now part of the package `depends`
-   field and the `dev-repo` and `homepage` point to your fork. It is also
-   recommended to remove the `doc` field to ease the rest of the release
-   process.
-3. Tag it with `X.Y.Z+dune`.
-4. Use `dune-release` to release your port to this repo as described in the
+The typical workflow to port an existing package `pkg`, version `X.Y.Z` to dune
+and publish it on `opam-overlays` would be the following:
+
+1. Create a fork of the upstream package `pkg` repo on dune-universe
+2. Create a branch `dune-universe-vX.Y.Z` pointing to the upstream `X.Y.Z`
```

Comment:
Some of our ports are `duniverse-X.Y.Z` i.e. without the dash and without the `v` (like e.g. extlib). I think the `v` should probably mirror the tags - if the tag has a `v` include it, if not then don't.

Reply:
I think consistency is better and we should always use the same thing, that way you don't even have to check for the tag.

Reply:
Yes, that's also a way to see it that would make sense to me as well.